### PR TITLE
Sorting upcoming meetings

### PR DESCRIPTION
**Pull request by fatcat2**
Now the next upcoming one is the first one

### CODE REVIEW

File: tests/test_meeting.py [hunk 1]
```diff
@@ -725,7 +725,7 @@ def test_meeting_get_upcoming_all(client):
         return_data = json.loads(rv.data.decode())
 
         assert return_data["access_token"] == "AccessToken"
-        assert len(return_data["data"]) == 3
+        assert len(return_data["data"]) == 1
```
Comment (stephend017) [inline tests/test_meeting.py:728]: why did this change?

**Review by stephend017 — APPROVED** [review]

**Review by DanielKambich — APPROVED** [review]
Thank you for requesting my review. I think this looks good, however, I am not as experienced with the backend so 🤷‍♂️

File: tests/test_meeting.py (hunk 1 above)
Comment (lzhahn) [reply tests/test_meeting.py:728]: I think this was changed to mark that only one meeting is returned from the call

File: biit_server/meeting_handler.py [hunk 2]
```diff
@@ -490,18 +490,24 @@ def meetings_get_upcoming(request, auth):
...
+        meeting
         for meeting in meetings
         if args["email"] in meeting.user_list
         and meeting.user_list[args["email"]] == 1
         and datetime.utcfromtimestamp(meeting.timestamp) > datetime.now()
     ]
 
+    meeting_dict = {meeting.id: meeting for meeting in filtered_meetings}
+    meeting_times = {meeting.id: meeting.timestamp for meeting in filtered_meetings}
+    sorted_meeting_times = sorted(meeting_times.items(), key=lambda kv: (kv[1], kv[0]))
```
Comment (lzhahn) [inline biit_server/meeting_handler.py:503]: Add a comment for what sorted sorts by

**Review by lzhahn — APPROVED** [review]
LGTM, just address the comments

File: tests/test_meeting.py (hunk 1 above)
Comment (fatcat2) [reply tests/test_meeting.py:728]: Changed what is getting tested

**Pull request merged**